### PR TITLE
records per page defaulted to 10 query param

### PR DESCRIPTION
**Pull request by choiii-darren**
#63

### CODE REVIEW

**Comment by ivanccheng** [comment]
gonna deploy to take a look

**Comment by choiii-darren** [comment]
woops i griefed i fixed now

File: front-end/src/components/Results.js [hunk 1]
```diff
@@ -125,13 +124,31 @@ const ResultsPage = () => {
...
+                            }}>5</Dropdown.Item>
+                            <Dropdown.Item key='10' value='10' onClick={(event) => {
+                                setRecordsPerPage(event.target.text)
+                            }}>10</Dropdown.Item>
+                            <Dropdown.Item key='15' value='15' onClick={(event) => {
+                                setRecordsPerPage(event.target.text)
+                            }}>15</Dropdown.Item>
+                            <Dropdown.Item key='20' value='20' onClick={(event) => {
+                                setRecordsPerPage(event.target.text)
+                            }}>20</Dropdown.Item>
```
Comment (ivanccheng) [inline front-end/src/components/Results.js:148]: should also update the active page, to 1 or do some math to get the current page based on the new per page

**Review by ivanccheng — CHANGES_REQUESTED** [review]
image
looks like the reason why it's returning a blank is bc the request is still requesting the old page num. 

flow: 
Default 10 per page, requests page 42
IF then Switches to 20 per page, but still requests 42, no data there. Should change the request page to reflect the new data per page

IMO an easy fix to this is simple refresh the route and set the page to 1 when `perPage` is changed, you can do whatever you like though

**Review by ivanccheng — APPROVED** [review]
Looks good to me!

nit/future improvement: 
We can add the "loading/retrieving data" text/modal when switching in between pages, to accomodate slower network connections or slower performance due to higher usage

**Comment by ivanccheng** [comment]
Remember to use squash and merge!

**Pull request merged**